### ISSUE TITLE
reverb reverts to original settings in reaper once fx window is closed

### ISSUE THREAD

**Issue by muziker** [michaelwillis/dragonfly-reverb#124]
In reaper 6.47 linux with dragonfly reverb 3.2.5-33-g438121a, the reverb selection when using the gui reverts back to the original initialised state once the fx gui is closed. In the generic UI editing view, it has also reverted to the original. The audio processing however seems to be the same, though all parameters are set to the init state.

**Comment by muziker** [comment]
For some reason in another project this doesn't happen.

**Comment by muziker** [comment]
This happened again. Once the gui is closed and another fx gui is open, returning back to the dragonfly gui changes the Level settings back to some default. I managed to work past it by saving and reopening the project again

**Comment by muziker** [comment]
Ok this is what happens for 3.2.6-16-g158ddaf. 

1. place Hall reverb into fx chain
2. change wander to 30 ms
3. defocus gui by clicking another plugin
4. return to dragonfly gui
5. the wander setting reverts to the original
6. But saving and rebooting the DAW, the wander setting is 30msec 

I haven't used it long enough to determine if it's a cosmetic issue or the settings do revert to the original as in item 5. In 3.2.5, it reverts to some previous version.

**Comment by Crono23** [comment]
I find that this happen on LV2 version of the plugin, the VSTs are fine.

**Comment by oiwn** [comment]
yes, can confirm, also noticed it. but i have it across all projects

v6.65 - August 12 2022 and vst3 versions of plugins.

**Comment by muziker** [comment]
From what i've experienced, this is a cosmetic change only. But if any of the controls are touched, then everything reverts to whatever is on screen. To test, once there seems to be a change, save everything in the daw and close. Reopen daw and observe if the settings are correct or the reverted default ones. This is pretty difficult to check because you'll have to remember the original state of the settings to begin with.

**Comment by michaelwillis** [comment]
@muziker I just got back to working on this after being away for a while. Please try [version 3.2.8](https://github.com/michaelwillis/dragonfly-reverb/releases/tag/3.2.8) and see if you can reproduce the bug.

**Comment by equiX-EAM** [comment]
The error persists in version 3.2.10 of Early Reflections. I use Ableton, and when I close the graphical interface, upon reopening it, the default preset is selected.
